src/Annotations/ValidatorException.java

```diff
@@ -0,0 +1,8 @@
+package Annotations;
+
+public class ValidatorException extends Exception {
+
+    public ValidatorException(String message) {
+        super(message);
```

Comment:
Try to implement logic of this method and write general error message for all ValidatorException instances.